tsconfig.json

```diff
@@ -12,6 +12,9 @@
     "forceConsistentCasingInFileNames": true,
     "strict": false,
     "skipLibCheck": true,
-    "sourceRoot": "./src"
-  }
+    "sourceRoot": "./src",
+  },
+  "include": [
```

Comment:
This set up is a bit weird: the packages are in `packages/dbadmin/`, resp. `src/`, but this `tsconfig.json` just finds everything. I think it'd be better to have a `core` and `dbadmin` package, and use the same setup as e.g. in the `lionweb-typescript` repository: one top-level `tsconfig.json`, and one per package that reuses the top-level one, but has a specific `sourceRoot` spec.

Reply:
I am not an expert in packages, but in the tutorial I read they were using a core module directly under `src` and then optional modules under packages. It seemed a bit asymmetric to me, but I thought it was the recommended standard. If that is not the case let's move to a core package

Reply:
Not an expert either, but I'm a bit worried about the apparent lack of separation between `src/` and `packages/` re: TS-compilation. I'm sure it works as long as source code is next to each other, though.